{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -63,7 +63,7 @@ repos:
   hooks:
       - id: mypy
         exclude: docs/|setup.py|tests/
-        args: [--no-strict-optional, --install-types, --non-interactive, --ignore-missing-imports, --disallow-untyped-defs]
+        args: ['--no-strict-optional', '--install-types', '--non-interactive', '--ignore-missing-imports', '--disallow-untyped-defs']
```

Comment:
as mentioned on slack, not sure its worth making this change.. (especially not in this PR)

Reply:
Change has been reverted to avoid introducing errors into PR